
<launch>

    <node name="forearm_taxels_viz" pkg="m3skin_viz"
        type="m3skin_viz" output='screen' ns='pr2_fabric_forearm_sensor'>
    </node>

    <node name="upperarm_taxels_viz" pkg="m3skin_viz"
        type="m3skin_viz" output='screen' ns='pr2_fabric_upperarm_sensor'>
    </node>

    <node name="gripper_right_link_taxels_viz" pkg="m3skin_viz"
        type="m3skin_viz" output='screen' ns='pr2_fabric_gripper_right_link_sensor'>
    </node>

    <node name="gripper_left_link_taxels_viz" pkg="m3skin_viz"
        type="m3skin_viz" output='screen' ns='pr2_fabric_gripper_left_link_sensor'>
    </node>

    <node name="gripper_palm_taxels_viz" pkg="m3skin_viz"
        type="m3skin_viz" output='screen' ns='pr2_fabric_gripper_palm_sensor'>
    </node>

    <!-- <node name="fingertip_taxels_viz" pkg="m3skin_viz" -->
    <!--     type="m3skin_viz" output='screen' ns='pr2_pps_sensor'> -->
    <!-- </node> -->

    <node name="fingertip_left_taxels_viz" pkg="m3skin_viz"
        type="m3skin_viz" output='screen' ns='pr2_pps_left_sensor'>
    </node>

    <node name="fingertip_right_taxels_viz" pkg="m3skin_viz"
        type="m3skin_viz" output='screen' ns='pr2_pps_right_sensor'>
    </node>

    <node name="ft_taxel_viz" pkg="m3skin_viz"
        type="m3skin_viz" output='screen' ns='pr2_ft_sensor'>
    </node>

</launch>

